applications/tari_base_node/src/command_handler.rs

```diff
@@ -95,44 +101,75 @@ impl CommandHandler {
         let peer_manager = self.peer_manager.clone();
         let mut connectivity = self.connectivity.clone();
         let mut metrics = self.dht_metrics_collector.clone();
+        let mut rpc_server = self.rpc_server.clone();
+        let config = self.config.clone();
 
         self.executor.spawn(async move {
+            let mut status_line = StatusLine::new();
+
             let state = state_info.recv().await.unwrap();
+            status_line.add_field("State", state.state_info.short_desc());
+
             let metadata = node.get_metadata().await.unwrap();
+
             let last_header = node
                 .get_headers(vec![metadata.height_of_longest_chain()])
                 .await
                 .unwrap()
                 .pop()
                 .unwrap();
-            let last_block_time: DateTime<Utc> = last_header.timestamp.into();
+            let last_block_time = DateTime::<Utc>::from(last_header.timestamp);
+            status_line.add_field(
+                "Tip",
+                format!(
+                    "{} ({})",
+                    metadata.height_of_longest_chain(),
+                    last_block_time.to_rfc2822()
+                ),
+            );
+
             let mempool_stats = mempool.get_mempool_stats().await.unwrap();
-            let banned_peers = banned_peers(&peer_manager).await.unwrap();
+            status_line.add_field(
+                "Mempool",
+                format!(
+                    "{}tx ({}g, +/- {}blks)",
+                    mempool_stats.total_txs,
+                    mempool_stats.total_weight,
+                    if mempool_stats.total_weight == 0 {
+                        0
+                    } else {
+                        1 + mempool_stats.total_weight / 19500
```

Comment:
**NIT:** I know this is historic and not part of the code changed in the PR, but is it not possible to use `ConsensusConstants::get_max_block_transaction_weight()` here instead of `19500`?

Reply:
Agree in principle, however, to do this will involve a minor refactor so I'd say it's out of scope for this PR.

Reply:
Agreed